src/entrypoints/is-array.d.ts

```diff
@@ -1,3 +1,12 @@
+type AsArray<T> = T[] extends T ? T[] : any[] 
```

Comment:
Could you inline this? Otherwise it'll be entered into the global scope.

Reply:
sure!